src/stacks-hierarchy/navigation-types.ts

```diff
@@ -48,6 +48,7 @@ type ReceiptScreenRouteParams = {
 
 type FareContractDetailsRouteParams = {
   orderId: string;
+  isSentFareContract?: boolean;
```

Comment:
Not sure exactly how this works, but is there a way to avoid send this param? Could FareContratDetails figure out itself if the fare contract is sent or not?

Reply:
We could, using `abtCustomerId` from `useAuthState()`, if we use that instead, we can remove all these parameter passing. 

The problem with `abtCustomerId` is that it can be undefined, and we will need to constantly check for its value on different screens/classes. 

Do you have any inputs/preference/best practice for this?

Reply:
I believe the performance cost of fetching the id from useAuthState is next to zero, so I think that's a better way of doing it. We should do whatever reduces complexity, but in general I think it's nice to have as few route params as possible.

Reply:
Updated 👍